devnet/docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+services: 
+  alice:
+    build:
+      context: ./..
+      dockerfile: devnet/alice.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
+        pubip: 10.5.0.2
+    ports:
+      - "7001"
+      - "8545"
+      - "8546"
+      - "8540"
+      - "9876"
+    networks:
+      vpcbr:
+        ipv4_address: 10.5.0.2
+
+  bob:
+    build:
+      context: ./..
+      dockerfile: devnet/bob.Dockerfile
+      args:
+        DD_API_KEY: ${DD_API_KEY}
```

Comment:
nit maybe/ideally we should load that variable from a .gitignore'd `.env` file since exporting in your shell will leave a trace in your shell history and this is an API key. Or not if we don't care, that works too :smile_cat:

Reply:
I don't think it matters that much for people working locally on their computer.  I need to ensure that we don't broadcast it on the github actions logs when I deploy to ECS.

Reply:
I think you can also use `secrets:` in docker-compose.yml, if that may be of use. [LINK](https://docs.docker.com/compose/compose-file/compose-file-v3/#secrets)